include/pmacc/mappings/kernel/MappingDescription.hpp

```diff
@@ -71,7 +71,7 @@ namespace pmacc
             {
                 /*minimal 3 blocks are needed if we have guarding blocks*/
                 int minBlock = std::min(gridSuperCells.x(), gridSuperCells.y());
-                if(DIM == DIM3)
+                if constexpr(DIM == DIM3)
                 {
                     minBlock = std::min(minBlock, gridSuperCells[2]);
```

Comment:
Now CI gives a warning/error `error: variable 'minBlock' set but not used [-Werror=unused-but-set-variable]` about line 73. Here we don't use variable `minBlock` at all, then this piece should be removed alltogether.

Reply:
I removed the change from this PR at all. The code part is "broken" and must be fixed. I will provide a fix in a seperate PR.